generators/client/templates/angular/src/main/webapp/app/core/user/user.model.ts.ejs

```diff
@@ -18,35 +18,35 @@
 -%>
 
 export interface IUser {
-    id?: any;
-    login?: string;
-    firstName?: string;
-    lastName?: string;
-    email?: string;
-    activated?: boolean;
-    langKey?: string;
-    authorities?: string[];
-    createdBy?: string;
-    createdDate?: Date;
-    lastModifiedBy?: string;
-    lastModifiedDate?: Date;
-    password?: string;
+id?: any;
+login?: string;
```

Comment:
The indentation was fine. Why the changes?

Reply:
seems to be some IntelliJ fuu